StanceClassifier/features/extract_features.py

```diff
@@ -28,19 +35,31 @@ def process_tweet_dict(self, tweet_dict):
             text = tweet_dict["text"]
         
         
-        tknzr = TweetTokenizer()
         FLAGS = re.MULTILINE | re.DOTALL
         text = re.sub(r"https?:\/\/\S+\b|www\.(\w+\.)+\S*", "http", text, flags=FLAGS)
         text = re.sub(r"@\w+", "@user", text, flags=FLAGS)
-        text_token = tknzr.tokenize(text)
+        text_token = self.nltk_tokenizer.tokenize(text)
```

Comment:
Minor optimization - I've moved the instantiation of the NLTK tokenizer into `__init__` so we only make one per run rather than one per call to the feature extractor.